packages/@aws-cdk/assertions/NOTICE

```diff
@@ -343,57 +343,6 @@ terms above.
 
 ----------------
 
-** lodash.clonedeep - https://www.npmjs.com/package/lodash
```

Comment:
Our linter was complaining that these attributions are no longer necessary...I checked that they are indeed not present in our dependencies. I'd venture this is some transitive dependency that no longer exists due to the jest upgrade.